webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,40 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+/* eslint-disable max-params  */
+
+export function notify(message, type, link, sticky) {
+  showToast({
+    message,
+    type,
+    sticky,
+    link
+  });
+}
+
+function showToast(toast) {
+  store.dispatch(ToastActions.addToast(toast));
+}
+
+// to accommodate rails flash syntax
+function importFlashMessagesFromRails() {
+  const notifications = $('#notifications').data().flash;
+
+  notifications.forEach(([type, message]) => {
+    // normalize rails flash names
+    if (type === 'danger') {
+      type = 'error';
+    }
+
+    showToast({
```

Comment:
Some plugins may still use `notify` in their js code, possibly with the wrong class, we don't want to break that ability or cause their notifications to brake. To me it also makes sense to have sticky default to `type!=='success'` if not otherwise specified, so that this is consistent across all notifications and not just those that arrive from rails.